src/zalopay/services/zod.ts

```diff
@@ -0,0 +1,104 @@
+import getJsonResponse from "../helpers/getJsonResponse";
+import {
+  ObjectSerializer,
+  ZODCreateInvoiceRequest,
+  ZODCreateInvoiceResponse,
+  ZODQueryInvoiceResponse,
+  ZODQueryStatusResponse
+} from "../models/models";
+import Service from "../service";
+import HmacUtils from "../utils/hmacUtils";
+import { ZaloPayClient } from "../zaloPayClient";
+import ZODResource from "./resource/zodResource";
+
+class ZOD extends Service {
```

Comment:
Class name should be Zod
We align this rule: https://google.github.io/styleguide/tsguide.html#identifiers-abbreviations